systemdev/qtmir-qtubuntu.rst

```diff
@@ -0,0 +1,55 @@
+QtMir and QtUbuntu
+==================
+
+`QtMir`_ and `QtUbuntu`_ are `Qt Platform Abstractions`_ (QPAs) for the Ubuntu Touch platform. This page is a collection of notes on debugging the QPAs and their related pieces.
+
+QtMir
+-----
+
+The `QtMir`_ QPA makes Qt a Mir server. This allows Unity8, the shell of Ubuntu Touch, to be written in QML and Qt C++ while operating on Mir Surfaces and Windows. It also allows the desktop to be represented and reasoned about as a `Qt Scene Graph`_.
```

Comment:
Does it really "make Qt a Mir server"? That sounds odd. Isn't it more like .... provides Qt bindings for Mir or exposes Mir API or smt like that?